components/file_upload_overlay.tsx

```diff
@@ -45,7 +45,6 @@ const FileUploadOverlay: React.FC<Props> = (props: Props) => {
                     <img
                         className='overlay__logo'
                         src={overlayLogoImage}
-                        width='100'
```

Comment:
Why did we remove width here?

Reply:
I passed it to the css selector, to improve the load time xD

Reply:
how were you able to measure this? i am curious

Reply:
@M-ZubairAhmed by using lighthouse you can audit errors in images for example. this was affecting the aspect ratio, making the browser recalculate the size:

<img width="682" alt="Screenshot 2022-12-30 at 21 33 15" src="https://user-images.githubusercontent.com/16692883/210106172-a1ea6650-53eb-487e-833d-bcc58f10337d.png">